client-src/default/index.js

```diff
@@ -9,6 +9,9 @@ const { log, setLogLevel } = require('./utils/log');
 const sendMessage = require('./utils/sendMessage');
 const reloadApp = require('./utils/reloadApp');
 const createSocketUrl = require('./utils/createSocketUrl');
+const updatePublicPath = require('./utils/updatePublicPath');
+
+updatePublicPath(__resourceQuery);
```

Comment:
I am afraid what we change global variable and it is can break code in some rare case, maybe we can rewrite this on using own variable like:
```
const resourceQuery = getPublicPath(__resourceQuery);
```

Reply:
Yes good point